Release_Protocol.md

```diff
@@ -14,9 +14,11 @@ $ git remote get-url upstream
 git@github.com:bids-standard/bids-specification.git
 ```
 
-If you do not, add it with:
+If you do not, clone the Master branch of the BIDS-specification and add it to `upstream` with:
 
 ```Shell
+$ git clone https://github.com/bids-standard/bids-specification.git
+$ cd bids-specification
```

Comment:
I usually add remotes using: `git remote add upstream https://github.com/bids-standard/bids-specification` --> for that I do not need to clone the upstream that I want to add.

I think @effigies wrote this part of the protocol, perhaps he can weigh in.

Reply:
Yeah, I don't think this is necessary. But we could add a clone to before the "You should have a remote", if it's not clear to people that they need to be in the repository. I assume most maintainers will already, though...

Reply:
I think adding that can be useful. I agree those that are charged with releasing will likely know this information already. Thought it may be helpful to have the protocol self-contained

Reply:
Okay, fine with me to include it. But wouldn't a developer/maintainer always clone their fork, rather than the upstream?

for me:
```shell
git clone https://github.com/sappelhoff/bids-specification.git
cd bids-specification
git remote add upstream https://github.com/bids-standard/bids-specification
```

Reply:
your path is what I took too. I think it is easier to git clone

Reply:
@franklin-feingold do you want to incorporate these changes in this PR? Or do you want to take this PR in another direction and we should do the revision somewhere else?

Reply:
what change may you be referencing? the change discussed above is incorporated - the git add upstream is the line below this comment. What you commented previously that was missing is what is added (the clone and cd). I did not change the `git@github.com:bids-standard/bids-specification.git` to `git remote add upstream https://github.com/bids-standard/bids-specification` because that is what Chris had. Can commit that change, I used Chris's process when processing the release (https://github.com/bids-standard/bids-specification/pull/291)

Reply:
Ah I see, yes that works - although in this first line `git clone https://github.com/bids-standard/bids-specification.git` the GitHub user `bids-standard` would usually be the username of the maintainer. I don't know if that **needs** adjustment, ... might be okay to just leave it as is for now.

Reply:
using `bids-standard` will ensure it is the most up to date. could be safe to assume the maintainers doing this process will have their fork synced but can be amended later if we find this is not needed

Reply:
@franklin-feingold I have made a PR to your PR-branch, see https://github.com/franklin-feingold/bids-specification/pull/27

I think it is better to mention a fork and a clone as the requirements for the release protocol, because the practical implementation of this may differ between maintainers/developers.

Reply:
merged in - thank you